plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/RepositoriesCMA.js

```diff
@@ -51,14 +51,6 @@ Ext.define('NX.coreui.controller.RepositoriesCMA', {
     'repository-default': {
       file: 'database_green.png',
       variants: ['x16', 'x32']
-    },
-    'repository-online': {
-      file: 'tick.png',
-      variants: ['x16']
-    },
-    'repository-offline': {
-      file: 'cross.png',
-      variants: ['x16']
```

Comment:
online/offline is different than the remote availability, shy are we removing this here?